db/db_test.go

```diff
@@ -2,45 +2,66 @@ package db
 
 import (
 	"fmt"
-	"testing"
 	"github.com/yoiner-castillo-globant/GBcamp/db"
+	"testing"
 )
 
 func TestCreate(t *testing.T) {
-	db.Create("test1", "50")
+	resp, err := db.Create("test1", "50")
+	if !resp {
```

Comment:
A gold rule, check first the error value instead the rest of returned values

Reply:
thank u